ingest/sqs/chain_info/usecase/chain_info_usecase.go

```diff
@@ -14,6 +13,15 @@ type chainInfoUseCase struct {
 	contextTimeout         time.Duration
 	chainInfoRepository    mvc.ChainInfoRepository
 	redisRepositoryManager mvc.TxManager
+
+	// N.B. sometimes the node gets stuck and does not make progress.
+	// However, it returns 200 OK for the status endpoint and claims to be not catching up.
+	// This has caused the healthcheck to pass with false positives in production.
+	// As a result, we need to keep track of the last seen height and time to ensure that the height is
+	// updated within a reasonable time frame.
```

Comment:
Can we have the GetLatestHeight function docs represent some of this information as well?

Reply:
[d18ff4c](https://github.com/osmosis-labs/osmosis/pull/7234/commits/d18ff4ce0bc8b7fe4c61e017cc4b96d640488471)